plenario/templates/explore.html

```diff
@@ -21,8 +22,8 @@
 {% block extra_javascript %}
     <script src="{{ url_for('static', filename='js/jquery-ui.min.js') }}"></script>
     <script src="{{ url_for('static', filename='js/jenks.js') }}" type="text/javascript"></script>
-    <script type="text/javascript" src="{{ url_for('static', filename='js/underscore-min.js') }}"></script>
```

Comment:
Oops! I switched out the minified JS libraries for the development versions for debugging. Please change back before we bring it into master.